app/src/main/java/com/android/sample/ui/workout/RunningSelectionScreen.kt

```diff
@@ -0,0 +1,110 @@
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.R
+import com.android.sample.ui.composables.TopBar
+import com.android.sample.ui.navigation.NavigationActions
+import com.android.sample.ui.theme.Black
+import com.android.sample.ui.theme.Blue
+import com.android.sample.ui.theme.NeutralGrey
+
+@Composable
+fun RunningSelectionScreen(navigationActions: NavigationActions) {
+  val buttonColor = ButtonDefaults.buttonColors(Blue)
+  Scaffold(
+      modifier = Modifier.testTag("RunningSelectionScreen"),
+      topBar = { TopBar(navigationActions, R.string.RunningWorkoutTitle) },
+      content = { pd ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(pd),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center) {
+              // Bouton "Without Path"
+              Button(
+                  onClick = { /* Action pour Without Path */},
+                  modifier =
+                      Modifier.fillMaxWidth(0.7f)
+                          .padding(vertical = 8.dp)
+                          .height(70.dp)
+                          .testTag("withoutPathButton")
+                          .shadow(8.dp, RoundedCornerShape(40.dp)),
+                  colors = buttonColor,
+                  shape = RoundedCornerShape(40.dp)) {
+                    Text(text = "Without Path", fontSize = 16.sp, color = Black)
+                  }
+
+              InterButtonSpacer()
+
+              // Bouton "Create New Path"
+              Button(
+                  onClick = { /* Action pour Create New Path */},
+                  modifier =
+                      Modifier.fillMaxWidth(0.7f)
+                          .padding(vertical = 8.dp)
+                          .height(70.dp)
+                          .testTag("createNewPathButton")
+                          .shadow(8.dp, RoundedCornerShape(40.dp)),
+                  colors = buttonColor,
+                  shape = RoundedCornerShape(40.dp)) {
+                    Text(text = "Create New Path", fontSize = 16.sp, color = Black)
+                  }
+
+              InterButtonSpacer()
+
+              // Bouton "Load Path"
+              Button(
+                  onClick = { /* Action pour Load Path */},
+                  modifier =
+                      Modifier.fillMaxWidth(0.7f)
+                          .padding(vertical = 8.dp)
+                          .height(70.dp)
+                          .testTag("loadPathButton")
+                          .shadow(8.dp, RoundedCornerShape(40.dp)),
+                  colors = buttonColor,
+                  shape = RoundedCornerShape(40.dp)) {
+                    Text(text = "Load Path", fontSize = 16.sp, color = Black)
+                  }
+
+              // Image de silhouette d'un coureur
```

Comment:
Really good UI !
But put the comment in English pls. Avoid to use two different languages

Reply:
Oui